src/Components/Onboarding/Steps/Artists/PopularArtists.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+
+import { artsyNetworkLayer } from "../../../../Relay/config"
+import PopularArtistQueryConfig from "../../../../Relay/Queries/PopularArtist"
+
+import SelectableItemContainer from "./SelectableItemContainer"
+
+export interface RelayProps {
+  popular_artists: {
+    artists?: any[]
+  }
+}
+
+export default class PopularArtists extends React.Component<null, null> {
+  render() {
+    return <PopularArtistContentList />
+  }
+}
```

Comment:
I don’t really understand why there are so many component classes with render methods that return a single component in this PR. Seems like there may be some unnecessary indirection.